dts/common/nordic/nrf54h20.dtsi

```diff
@@ -584,6 +584,13 @@
 				interrupts = <300 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
 
+			egu130: rtc@929000 {
```

Comment:
```suggestion
			egu130: rtc@92d000 {
```

Please add also a corresponding entry in [validate_base_addresses.c](https://github.com/zephyrproject-rtos/zephyr/blob/fd0e46506fabd4c0c91ecb336a39cb6a6aa2b270/soc/nordic/validate_base_addresses.c#L176).